### ISSUE TITLE
gather: behavior when collecting dtmf with no interdigit timeout

### ISSUE THREAD

**Issue by davehorton** [jambonz/jambonz-feature-server#107]
When doing a gather with dtmf input, if you specify a minDigits and maxDigits, but you do *not* specify an interDigitTimeout, and then the caller enters a number of digits in the {minDigits, maxDigits} range and then pauses indefinitely, what happens is that the overall timeout is reached and the actionHook returns reason: timeout.

What actually should happen in this case is that even though we did reach the timeout, since we have collected at least minDigits, the actionHook should return those digits and reason 'dtmf-num-digits'.

In other words, there is no default for interDigitTimeout but if we reach the overall timeout and have collected a valid set of digits we should return them

**Comment by xquanluu** [comment]
Hello @davehorton ,

I see this case is already resolved here:
https://github.com/jambonz/jambonz-feature-server/blob/main/lib/tasks/gather.js#L386

Is this task is still valid?

**Comment by davehorton** [comment]
yes it has been fixed
